pkgs/tools/text/ocrmypdf/default.nix

```diff
@@ -50,6 +50,7 @@ buildPythonApplication rec {
     chardet
```

Comment:
```suggestion
```

Reply:
Thanks for the suggestion - applied & squashed.